src/routes/user.js

```diff
@@ -11,5 +17,6 @@ router.post('/', create)
 router.get('/', validateAuthentication, getAll)
 router.get('/:id', validateAuthentication, getById)
 router.patch('/:id', validateAuthentication, validateTeacherRole, updateById)
+router.patch('/update/:id', validateAuthentication, updateProfile)
```

Comment:
Can you explain the difference between the PATCH request above `/:id` and the new one you added `/update/id:`?